src/mscorlib/System.Private.CoreLib.csproj

```diff
@@ -445,6 +444,7 @@
   <ItemGroup>
     <Compile Include="$(BclSourcesRoot)\Internal\Runtime\Augments\EnvironmentAugments.cs" />
     <Compile Include="$(BclSourcesRoot)\Internal\Runtime\Augments\RuntimeThread.cs" />
+    <Compile Include="$(BclSourcesRoot)\Internal\Console.cs" />
```

Comment:
I have moved it to internal namespace instead. It makes switching between the internal version and the official version a bit easier.